@xen-orchestra/lite/src/lang/en.json

```diff
@@ -1,19 +1,27 @@
 {
+  "availableUpdates": "{nUpdates} available update{nUpdates, plural, one {} other {s}}",
   "badCredentials": "Bad credentials",
   "confirmCtrlAltDel": "Send Ctrl+Alt+Del to VM?",
   "connect": "Connect",
   "consoleNotAvailable": "Console is only available for running VMs",
   "ctrlAltDel": "Ctrl+Alt+Del",
+  "description": "Description",
   "device": "Device",
   "disconnect": "Disconnect",
   "dns": "DNS",
+  "errorOccurred": "An error has occurred. Please try again later.",
```

Comment:
If we want to be able to reuse that message, maybe don't say that they should "try again later" since it might not be relevant in other cases.